terraform/groups/ecs-service/profiles/development-eu-west-2/cidev/vars

```diff
@@ -15,6 +15,7 @@ landing_page_url = "/register-an-overseas-entity/sold-land-filter?start=0"
 landing_page_starting_new_url = "/register-an-overseas-entity/starting-new"
 payment_fee = "100"
 update_landing_page_url = "/update-an-overseas-entity/continue-with-saved-filing"
+remove_landing_page_url = "/gov-uk-remove-guidance-page"
```

Comment:
i thought the landing page would be /update-an-overseas-entity/continue-with-saved-filing?journey=remove and that /gov-uk-remove-guidance-page would be the 'back' link from the landing page

Reply:
I think we clarified this in discussion yesterday - I've now made these static HTTP URLs that don't 'point into' our web app journey. So a bit more like the config in LIVE.